src/features/serde/mod.rs

```diff
@@ -59,6 +59,8 @@ mod de_borrowed;
 mod de_owned;
 mod ser;
 
+use core::fmt::Debug;
```

Comment:
I don't think this import is used because you reference all the implementations directly.

If you want to keep this import, please put it below the `pub use`